rpm/0001-aes-asm-aesv8-armx.pl-20-improvement-on-ThunderX2.patch

```diff
@@ -0,0 +1,464 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
I get the changes to the patch header etc. but is there a reason to rename the patch here from `openssl-1.1.1-arm-update.patch`? It doesn't now seem to fit the naming convention of the other patches.

Reply:
It is because it is a different patch technically, I cherry-pick the the change from the 3.0 and rebased it my self.
I can add that to the commit message.

Reply:
I agree it would be better if the name of the patch was aligned with the existing ones. maybe something like openssl-3.x-backport-etc would be more descriptive? There's not a lot of point having the 0001.

Reply:
That's what comes back from git, all other patches have "openssl-<version>" as prefix is maybe better.

Reply:
git needs the numbers to know which order to re-apply the patches, but that's not the case here (in this sense it's a shame all of them aren't numbered, but that's a separate issue). Any reason not to go with your "openssl-" suggestion?